.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /hook/target
+/.idea
```

Comment:
Nit: this should probably go into your local `~/.gitconfig_global` or something like that (also missing newline).

Reply:
Adding common build tools to `.gitignore` is fine